test/runnable/inline.d

```diff
@@ -776,10 +776,30 @@ void test7625()
 {
     int x = foo7625(1);
     if (x != 1)
-	assert(0);
+        assert(0);
 }
 
 /**********************************/
+// 9785 partial fix
+
+void test9785()
+{
+        int j = 3;
+
+        void loop(scope const void function(int x) dg) {
```

Comment:
Doing the whole thing is too complex for one pull. This is the simplest case, and I wanted to verify the scaffolding all works before making it complex.